service/job/jobs.py

```diff
@@ -188,6 +190,71 @@ def _create_chains(self, params, user_name):
 
         return chains
 
+    def _create_pdf_metadata(self, metadata):
```

Comment:
Ich finde die Methode sollte nicht in der jobs.py stehen. Erstens braucht sie, soweit ich das sehen kann, keinen Zugriff auf die Instanz von IngestArchivalMaterialsJob. Zweitens sollten in der Datei aus meiner Sicht ausschließlich Definitionen von Job-Chains enthalten.

Reply:
Dann eine Funktion in einer eigenen Datei, der man job type und metadaten übergibt? 

Auf Dauer würde ich es so sehen wie du: Das bläht die jobs.py vermutlich auf.

Ich hätte aber eher gedacht, dass man dann ggf. irgendwann pro Jobtyp eine eigene Datei/Klasse hat, die die ganzen Methoden definiert (im Gegensatz zu deiner Idee mit nur Chaindefinitionen in der jobs.py).